packages/plugin/src/protobufts-plugin.ts

```diff
@@ -97,6 +97,10 @@ export class ProtobuftsPlugin extends PluginBase {
                 "the default behaviour, this option has no effect.",
             excludes: ['eslint_disable'],
         },
+        force_disable_services: {
+            description: 'Do not generate GRPC services, clients, or servers.',
```

Comment:
We should give a bit more details about what this option does, for example:

```suggestion
            description: "Do not generate anything for service definitions, and \n" +
                         "ignore options in proto files. This is the same as setting both \n" + 
                         "`force_server_none` and `force_client_none`, but also stops \n" + 
                         "generating service metadata."
```